app/controllers/carto/api/public/custom_visualizations_controller.rb

```diff
@@ -0,0 +1,144 @@
+require_relative '../visualization_searcher'
+require_relative '../paged_searcher'
+
+class Carto::Api::Public::CustomVisualizationsController < Carto::Api::Public::ApplicationController
+  include Carto::Api::VisualizationSearcher
+  include Carto::Api::PagedSearcher
+
+  CONTENT_LENGTH_LIMIT_IN_BYTES = 20000
```

Comment:
We need to increase this value, maybe to 100kb or 200kb 
```suggestion
  CONTENT_LENGTH_LIMIT_IN_BYTES = 100000
```